test/integration/ads_integration_test.cc

```diff
@@ -150,7 +150,7 @@ class AdsIntegrationTest : public HttpIntegrationTest,
   }
 
   void initialize() override {
-    BaseIntegrationTest::initialize();
+    initialized_ = true;
```

Comment:
This seems a bit more fragile than before. Can we wrap this in a `setInitialized()` or `markInitializaitonComplete()` setter?

Reply:
How about a TODO to port the rest of the tests over and make initialize() non-virtual?  I have a follow-up PR which converts 5 of the 7 already :-)

Reply:
I'm a little confused now. Why do we need to call initialize() sometimes and not others?

Reply:
Some tests don't (yet) want the base class initialize (they use old style json files) so they override.  They set initialized_ true for consistency.